src/main/java/com/thebois/views/toolbar/StructureToolbarView.java

```diff
@@ -0,0 +1,60 @@
+package com.thebois.views.toolbar;
+
+import com.badlogic.gdx.scenes.scene2d.Actor;
+import com.badlogic.gdx.scenes.scene2d.ui.ButtonGroup;
+import com.badlogic.gdx.scenes.scene2d.ui.Skin;
+import com.badlogic.gdx.scenes.scene2d.ui.Table;
+
+import com.thebois.listeners.IEventListener;
+import com.thebois.listeners.events.OnClickEvent;
+import com.thebois.models.world.structures.StructureType;
+import com.thebois.views.game.StructureButton;
+import com.thebois.views.info.IActorView;
+
+/**
+ * View that renders structure buttons used for selecting what structure to build.
+ */
+public class StructureToolbarView implements IActorView {
```

Comment:
style

This is more of a `BuildMenuToolbarView`, right? It's not a view of structures, it's a view of a menu that allows for selecting of which structure to build.